### PR TITLE
🏗️ Adjusting requests and response for TOTP flow.

### PR DESCRIPTION
**Pull request by Danieloni1**

### CODE REVIEW

File: requests.go [hunk 1]
```diff
@@ -92,8 +94,8 @@ type ReqPutAccountData struct {
 	Data map[string]interface{}
 }
 
-// ReqEmailRequestToken is the JSON request for
-// 		https://matrix.org/docs/spec/client_server/r0.6.1#post-matrix-client-r0-register-email-requesttoken
+//	 ReqEmailRequestToken is the JSON request for
```
Comment (PiotrKozimor) [inline requests.go]: Was it formatted automatically?

**Review by PiotrKozimor — CHANGES_REQUESTED** [review]

File: requests.go (hunk 1 above)
Comment (PiotrKozimor) [reply requests.go]: https://tip.golang.org/doc/go1.19#go-doc might help

Comment (Danieloni1) [reply requests.go]: @PiotrKozimor 
If I just save the file without changing anything there, `gofmt` turns the comments into this:
image
But if I add a single space then it turns to this here 😓 
Whatever you prefer 😆

Comment (PiotrKozimor) [reply requests.go]: This one looks better to me :)

Comment (Danieloni1) [reply requests.go]: @PiotrKozimor Done :)

File: requests.go [hunk 2]
```diff
@@ -22,6 +22,9 @@ type ReqLogin struct {
 	Token                    string     `json:"token,omitempty"`
 	DeviceID                 string     `json:"device_id,omitempty"`
 	InitialDeviceDisplayName string     `json:"initial_device_display_name,omitempty"`
+	InhibitLogin             bool       `json:"inhibit_login"`
```
Comment (PiotrKozimor) [inline requests.go]: `inhibit_device`, right?

Comment (Danieloni1) [reply requests.go:25]: I figured both are indicative, but on a second thought, `inhibit_device` might help differentiate between `register` `inhibit_login` and this `inhibit_device`.

Comment (Danieloni1) [reply requests.go]: Done

Comment (Danieloni1) [reply requests.go]: @PiotrKozimor 
Done

**Review by PiotrKozimor — APPROVED** [review]

**Pull request merged**